roles/nginxplus/files/ssl/lib-sc-prod_princeton_edu_chained.pem

```diff
@@ -1,41 +1,41 @@
-VzxT8MDkOHhwJQgXL6OqHQcT0wwAAAF5GzOn7AAABAMASDBGAiEA1GxU9xi3f+By
-+fatnQi/BOLBHfucYoJZIZFuhwaKpggCIQCHiCnvWqh3R1MmjAzxIctzaAMtAWB1
-w8aySgMMcE2aCTANBgkqhkiG9w0BAQsFAAOCAQEAkPWOzWQV0HPsXXenYwJ+TMvA
-uzG1lZsxoEcRoOSK2vpEQ5n8GmZ2UxMR9WZcq20E5TcQehbF1X3KZ5vwHXXpr/GD
-7K9bmFLi2Io5Zd9f9Dv0RmtnkV3aEOEFnJkCo7lKaOdFmBQlaGMJokUxOLPbrhlq
-AZPw9fstOgiPqii1UH34R3k8Fajtpt4WQWfL3+Fyx7uXn9BblfK/1+hYyEjgxV38
-0QpBZ6XfitSMKyjcp6iOA4Oymmq3lM58N+UEz805DlCh7q3eGRxJWnKhAFA9Sc34
-CTM2W2XAM5HPf1dNTTBPSN8KJKUzFp1GcAUr1tCqcp0YfO11MBNJCLIDFkdv3w==
+MIIG1zCCBb+gAwIBAgIRAIPWtdaSpDQaXFdU8za00KUwDQYJKoZIhvcNAQELBQAw
+djELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAk1JMRIwEAYDVQQHEwlBbm4gQXJib3Ix
+EjAQBgNVBAoTCUludGVybmV0MjERMA8GA1UECxMISW5Db21tb24xHzAdBgNVBAMT
+FkluQ29tbW9uIFJTQSBTZXJ2ZXIgQ0EwHhcNMjIwNTAzMDAwMDAwWhcNMjMwNjAz
+MjM1OTU5WjCBgzELMAkGA1UEBhMCVVMxEzARBgNVBAgTCk5ldyBKZXJzZXkxLTAr
+BgNVBAoTJFRoZSBUcnVzdGVlcyBvZiBQcmluY2V0b24gVW5pdmVyc2l0eTEMMAoG
+A1UECxMDT0lUMSIwIAYDVQQDExlsaWItc2MtcHJvZC5wcmluY2V0b24uZWR1MIIB
+IjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA3QOkUfW2jqdpdO9jMj4xg0HX
+nqUGN53D1PrnuzzzcCeskrsqoqd2xkg6jzrrOB5WX5smv+dTwFaZsb/Z72o7cJT2
+Ywx886udIECuIL22a61sCmiZlN/qG7SlN82x8Oye5W+n5MvvHFuodRUMfjia+eeA
+ByJflBUtMK3gNxftcOup8AxfrFBdIi6pDh8d/f9EP40Pk9mHe5cxRIGvshDhQYbI
+ph7rjtFCVYTDmZWOm9J1b5NuL2DQ5oy0AdW3kISKfLnzDRBQyMRgdgYZ6WrRKkc9
+ngzTneievvM1S1I9cU/pVTXkxMZDT0tcPbAJlSClekBY+zOacTMkj7PhmF1YlQID
+AQABo4IDUDCCA0wwHwYDVR0jBBgwFoAUHgWjd49sluJbh0umtIascQAM5zgwHQYD
+VR0OBBYEFAt5UCqBdnm7uLp4FocHydczcSMMMA4GA1UdDwEB/wQEAwIFoDAMBgNV
+HRMBAf8EAjAAMB0GA1UdJQQWMBQGCCsGAQUFBwMBBggrBgEFBQcDAjBnBgNVHSAE
+YDBeMFIGDCsGAQQBriMBBAMBATBCMEAGCCsGAQUFBwIBFjRodHRwczovL3d3dy5p
+bmNvbW1vbi5vcmcvY2VydC9yZXBvc2l0b3J5L2Nwc19zc2wucGRmMAgGBmeBDAEC
+AjBEBgNVHR8EPTA7MDmgN6A1hjNodHRwOi8vY3JsLmluY29tbW9uLXJzYS5vcmcv
+SW5Db21tb25SU0FTZXJ2ZXJDQS5jcmwwdQYIKwYBBQUHAQEEaTBnMD4GCCsGAQUF
+BzAChjJodHRwOi8vY3J0LnVzZXJ0cnVzdC5jb20vSW5Db21tb25SU0FTZXJ2ZXJD
+QV8yLmNydDAlBggrBgEFBQcwAYYZaHR0cDovL29jc3AudXNlcnRydXN0LmNvbTAk
+BgNVHREEHTAbghlsaWItc2MtcHJvZC5wcmluY2V0b24uZWR1MIIBfwYKKwYBBAHW
+eQIEAgSCAW8EggFrAWkAdwCt9776fP8QyIudPZwePhhqtGcpXc+xDCTKhYY069yC
+igAAAYCKb4seAAAEAwBIMEYCIQCVPgtFscbk5p1vP247c2gDryqhJ8O71VTMiexK
+ms0EqQIhAN6UwnrgQpuZ2Y3DRxT1X9vgkI0lhktU/JHQKavjb88fAHYAejKMVNi3
+LbYg6jjgUh7phBZwMhOFTTvSK8E6V6NS61IAAAGAim+K3gAABAMARzBFAiALIKEU
+VnzxasxNXlvzCXwhFniCFvUzBCmfA0WXaAhapQIhAPnz7n5GlIsds+KEKpaPPk9d
+V51obps+LTZWJoQU1YvOAHYA6D7Q2j71BjUy51covIlryQPTy9ERa+zraeF3fW0G
+vW4AAAGAim+KsgAABAMARzBFAiEA6/d4vklbhtb5mS8gsLtUfNX5CCD+b2zkNqUt
+/G8VRlwCIHQGoP5mU5qmfPSctDC9SK6HXu8an2kNshm/McpQ0MpwMA0GCSqGSIb3
+DQEBCwUAA4IBAQA1ylHQlaRjlFm0zpoJlh28krJ4HaGyKcPrbUa28YW30zdGeE9P
+ZFY5+KGadPSOg6Hw9EQfQ3Hgp33fl5JmXH09RoezNo/HpHmwgDSvI8sAjFuIo7JH
+lDsfiVd5BPuZou/t4L8eT/GshLyr85vc1DGwQzs7b8eiBc4BexP5VvBtaKKvt51P
+2L4Ot2ImHBqtg28POvNJGn+szloChXg9f6KhIDYoRgz3buVTx4dI+FS1lAebJn5T
+JXrY0/xkXb8Fza8Y1q7LNltGXBAH1aTuKN1+S0IMHIM8ru+qMlTeszuLWZpk4qIA
+6asquhGDNCjqIkcj/h0P5fldfJwiTscZOpFk
 -----END CERTIFICATE-----
```

Comment:
Shouldn't there be changes to this part of the chained cert too?

Reply:
```
❯ openssl x509 -in lib-sc-prod_princeton_edu_chained.pem -text
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            83:d6:b5:d6:92:a4:34:1a:5c:57:54:f3:36:b4:d0:a5
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: C=US, ST=MI, L=Ann Arbor, O=Internet2, OU=InCommon, CN=InCommon RSA Server CA
        Validity
            Not Before: May  3 00:00:00 2022 GMT
            Not After : Jun  3 23:59:59 2023 GMT
        Subject: C=US, ST=New Jersey, O=The Trustees of Princeton University, OU=OIT, CN=lib-sc-prod.princeton.edu
```

Reply:
I don't know how to answer that... but it checks out.

Reply:
Working code wins.